### PR TITLE
 Self-updating build string does not work on Windows #697 

### PR DESCRIPTION
**Pull request by Quintinus98**
**Description**
It's useful to see in the "Overview" tab when ChRIS_ui was last updated, so we use a bash script to print git information and the date at build-time. This script doesn't work on Windows. It is more elegant if we can achieve a pure node.js project for cross-platform compatibility and perhaps to better benefit from static code analysis tools.

**The Fix**
Translated the print_version.sh file from bash to pure node.js, and using cross-platform commands such as git and npm only.
Screenshot from 2022-10-25 06-46-46

**Outcome**
The _Build_ syntax on the Overview page as seen, now works on pure nodejs and is cross-platform compatible.

**Notes**
Removed:
- print_version.sh file
- print_version.sh script in package.json

**Original Issue**
This PR fixes issue #697

### CODE REVIEW

**Comment by ciradu2204** [comment]
@jennydaman, we are waiting for your review. Thank You

File: src/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -56,10 +55,40 @@ const DashboardPage = (props: DashboardProps) => {
...
+      const today = new Date();
+      const yyyy = today.getFullYear().toString();
+      const mm = String(today.getMonth() + 1).padStart(2, '0');
+      const dd = String(today.getDate()).padStart(2, '0');
+      return [yyyy, mm, dd].join("");
+    }
+
+    const diff = () => {
+      // Run external tool synchronously
+      const result = shell.exec('git diff --quiet src/').code
```
Comment (jennydaman) [inline src/pages/Dashboard/Dashboard.tsx:72]: Prefer not to use dependency if the standard library function is easy enough to use.

```javascript
child_process.spawnSync('git', ['diff', '--quiet', 'src/']).status
```

**Review by jennydaman — CHANGES_REQUESTED** [review]

File: src/pages/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (Quintinus98) [reply src/pages/Dashboard/Dashboard.tsx:72]: Thanks for the update, I will make the requested changes

**Comment by Quintinus98** [comment]
@jennydaman I have made the requested changes. Could you have a look at it and merge the PR if it meets your requirements.
Thank you.

**Pull request closed without merging**